tests/functional/WebDriverTestCase.php

```diff
@@ -139,7 +139,12 @@ public static function skipForJsonWireProtocol($message = 'Not supported by Json
      */
     protected function getTestPageUrl($path)
     {
-        return 'http://localhost:8000/' . $path;
+        $host = 'http://localhost:8000';
+        if ($alternateHost = getenv('FIXTURES_HOST')) {
+            $host = $alternateHost;
+        }
+
+        return "{$host}/{$path}";
```

Comment:
We usually use this style of concatenation through the codebase:

```suggestion
        return $host . '/' . $path;
```

I will update it and merge the PR :)